src/Tizen.NUI.Components/Controls/Navigation/Page.cs

```diff
@@ -57,8 +57,23 @@ public class PageDisappearedEventArgs : EventArgs
     /// <since_tizen> 9 </since_tizen>
     public abstract class Page : Control
     {
+        private static readonly int DefaultTransitionDuration = 500;
```

Comment:
please check the VS warning below.

**src/Tizen.NUI.Components/Controls/Navigation/Page.cs#L60
Field 'DefaultTransitionDuration' is declared as 'readonly' but is initialized with a constant value. Mark this field as 'const' instead.**

Reply:
Thank you I fixed.